shared/actions/chat/index.js

```diff
@@ -775,22 +775,6 @@ function * _changedFocus (action: ChangedFocus): SagaGenerator<any, any> {
 
 function * _badgeAppForChat (action: Constants.BadgeAppForChat): SagaGenerator<any, any> {
   const conversations = action.payload
-  const selectedConversationIDKey = yield select(Constants.getSelectedConversation)
```

Comment:
🔥  we should just use the value we get from the daemon. This 'filtered' view just masks actual problems with the badge count